From ce804875914f52dba5356f8aefec850d411b051a Mon Sep 17 00:00:00 2001
From: Chris Wilson <chris@chris-wilson.co.uk>
Date: Thu, 31 Jul 2014 15:18:02 +0000
Subject: i810: Fix startup after marking GPU screens in the module

Fixes regression from

commit d6830190bc9956f485db6c3420f7a4a79793a693
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Wed Jul 23 10:53:40 2014 +0100

    sna: Disable all outputs when initializing as a slaved output provider

Reported-by: Tobias Powalowski
Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
---
diff --git a/src/legacy/i810/i810_driver.c b/src/legacy/i810/i810_driver.c
index a1b73fd..3d53d04 100644
--- a/src/legacy/i810/i810_driver.c
+++ b/src/legacy/i810/i810_driver.c
@@ -152,7 +152,7 @@ static int i810_pitches[] = {
 static Bool
 I810GetRec(ScrnInfoPtr scrn)
 {
-   if (((uintptr_t)scrn->driverPrivate & 1) == 0)
+   if (((uintptr_t)scrn->driverPrivate & 3) == 0)
       return TRUE;
 
    scrn->driverPrivate = xnfcalloc(sizeof(I810Rec), 1);
--
cgit v0.9.0.2-2-gbebe
